### PR TITLE
add payment

### PR DESCRIPTION
**Pull request by ShreeChander**
added payment page

### CODE REVIEW

File: src/main/webapp/payment.html [hunk 1]
```diff
@@ -5,10 +5,150 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Payment</title>
+  <title>Login jBookStall</title>
```
Comment (SU-Ki-MA) [inline src/main/webapp/payment.html]: Change Title to Payment

File: src/main/webapp/payment.html [hunk 2]
```diff
@@ -5,10 +5,150 @@
...
+	<path d="M 73.658 55.457 c -1.58 0 -2.974 0.734 -3.908 1.862 c -0.935 -1.128 -2.329 -1.862 -3.908 -1.862 c -2.814 0 -5.096 2.282 -5.096 5.096 c 0 2.814 2.282 5.096 5.096 5.096 c 1.58 0 2.974 -0.734 3.908 -1.862 c 0.935 1.128 2.329 1.862 3.908 1.862 c 2.814 0 5.096 -2.282 5.096 -5.096 C 78.754 57.738 76.472 55.457 73.658 55.457 z" style="stroke: none; stroke-width: 1; stroke-dasharray: none; stroke-linecap: butt; stroke-linejoin: miter; stroke-miterlimit: 10; fill: rgb(84,92,86); fill-rule: nonzero; opacity: 1;" transform=" matrix(1 0 0 1 0 0) " stroke-linecap="round" />
+	<path d="M 70.668 64.649 c 0.838 0.622 1.865 0.999 2.99 0.999 c 2.814 0 5.096 -2.282 5.096 -5.096 c 0 -1.064 -0.328 -2.05 -0.885 -2.867 C 75.889 60.401 73.454 62.762 70.668 64.649 z" style="stroke: none; stroke-width: 1; stroke-dasharray: none; stroke-linecap: butt; stroke-linejoin: miter; stroke-miterlimit: 10; fill: rgb(73,79,74); fill-rule: nonzero; opacity: 1;" transform=" matrix(1 0 0 1 0 0) " stroke-linecap="round" />
+</g>
+</svg>
+    
+    <div class="txt">
+    
+      <form class="needs-validation" novalidate>
+        <!-- <div class="row">
```
Comment (SU-Ki-MA) [inline src/main/webapp/payment.html]: remove code-comments

**Review by SU-Ki-MA — CHANGES_REQUESTED** [review]
Perform the mentioned changes.

File: src/main/webapp/payment.html (hunk 1 above)
Comment (ShreeChander) [reply src/main/webapp/payment.html]: Ok

File: jBookStall.sql [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+create table books (b_id varchar(11) primary key,
+ b_name varchar(50) not null,
+ b_price DOUBLE(10,2) not null,
+ b_stock int(11) not null,
+ b_copy_sold int(11) not null,
+ a_id varchar(50) not null,
+ pages int not null,
+ book_year year(4),
+ act_creation_time datetime,
```
Comment (SU-Ki-MA) [inline jBookStall.sql]: act_creation_time -> creation_time

File: jBookStall.sql [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+last_access_time datetime,
+session_creation datetime,
+session_ip_address varchar(50) not null,
+session_id varchar(50) not null,
+
+ FOREIGN KEY (a_id) REFERENCES authors(a_id) );
+ 
+create table users(u_id varchar(50) primary key , uname varchar(70) not null , email varchar(50) not null UNIQUE, pass_hash varchar(50) not null, salt varchar(50) not null,
```
Comment (SU-Ki-MA) [inline jBookStall.sql]: pass_hash increase size 256+, remove salt

File: jBookStall.sql (hunk 3 above)
Comment (SU-Ki-MA) [reply jBookStall.sql]: move every attribute related to session, to user table

**Review by SU-Ki-MA — CHANGES_REQUESTED** [review]
[x] payment.html
[ ] jBookStall.sql

perform mentioned changes for jBookStall.sql file

**Review by SU-Ki-MA — APPROVED** [review]

**Pull request merged**